README.md

```diff
@@ -1,2 +1,46 @@
 # comp110-worksheet-B
 Base repository for COMP110 worksheet B
+
+## The aim of this pseudocode is to show the process of selecting a secret word from a string of other words. Which has been selected by another user. 
+
+ 
+
+attemptNo � 4 
+
+wordLength � Number of letter in the word 
+
+secretWord � The secret word input 
+
+success � False 
+
+ 
+
+print (�Please guess the�, wordLength, �letter word.�) 
+
+ 
+
+while the attempt value is > 0 AND success = False 
```

Comment:
It is not clear where the `while` loop ends -- use indentation and/or something like `end while` to make this clearer